lib/reservation_manager.rb

```diff
@@ -0,0 +1,111 @@
+require 'date'
+require_relative 'reservation_dates'
+require_relative 'reservation'
+
+module HotelBookings
+  class Reservation_Manager
```

Comment:
Nitpick: the Ruby naming convention would call this `ReservationManager`, without the `_`